cmd/xl-v1-healing.go

```diff
@@ -138,7 +138,7 @@ func xlShouldHeal(partsMetadata []xlMetaV1, errs []error) bool {
 			return true
 		}
 		if errs[index] != nil {
-			continue
+			return true
```

Comment:
oh, no, xlShouldHeal should have been:
```
func xlShouldHeal(partsMetadata []xlMetaV1, errs []error) bool {                                                                                                                
        modTime := commonTime(listObjectModtimes(partsMetadata, errs))                                                                                                          
        for index := range partsMetadata {                                                                                                                                      
                if errs[index] == errDiskNotFound {                                                                                                                             
                        continue                                                                                                                                                
                }                                                                                                                                                               
                if errs[index] != nil {                                                                                                                                         
                        return true                                                                                                                                             
                }                                                                                                                                                               
                if modTime != partsMetadata[index].Stat.ModTime {                                                                                                               
                        return true                                                                                                                                             
                }                                                                                                                                                               
        }                                                                                                                                                                       
        return false                                                                                                                                                            
}   
```

i.e any error other than errDiskNotFound should be means it needs healing.